react/Shelf.js

```diff
@@ -123,12 +106,16 @@ const options = {
   }) => ({
     ssr: true,
     variables: {
-      ...(category != null ? {
-        category: category.toString(),
-      } : {}),
-      ...(collection != null ? {
-        collection: collection.toString(),
-      } : {}),
+      ...(category != null
+        ? {
+            category: category.toString(),
+          }
+        : {}),
+      ...(collection != null
+        ? {
+            collection: collection.toString(),
+          }
+        : {}),
```

Comment:
Remove the line break before each `category:`and format your file. Prettier should inline those objects 🤙